stories/core/Tree.stories.tsx

```diff
@@ -0,0 +1,139 @@
+  },
+  title: 'Core/Tree',
+} as Meta<TreeProps<unknown>>;
+
+export const Basic: Story<TreeProps<unknown>> = () => {
+  type StoryData = {
+    id: string;
+    label: string;
+    subLabel: string;
+    subItems: StoryData[];
+  };
+
+  const [selectedNodes, setSelectedNodes] = React.useState([
+    'Node 0',
+    'Node 3.2',
+    'Node 22',
+  ]);
+  const onSelectedNodeChange = React.useCallback(
+    (nodeId: string, selected: boolean) => {
+      if (selected) {
+        setSelectedNodes((oldSelected) => [...oldSelected, nodeId]);
+        action(`Selected node ${nodeId}`)();
+      } else {
+        setSelectedNodes((oldSelected) =>
+          oldSelected.filter((item) => item != nodeId),
+        );
+        action(`Unselected node ${nodeId}`)();
+      }
+    },
+    [],
+  );
+
+  const [expandedNodes, setExpandedNodes] = React.useState([
+    'Node 2',
+    'Node 2.1',
+    'Node 3',
+  ]);
+  const onNodeExpanded = React.useCallback(
+    (nodeId: string, expanded: boolean) => {
+      if (expanded) {
+        setExpandedNodes((oldExpanded) => [...oldExpanded, nodeId]);
+        action(`Expanded node ${nodeId}`)();
+      } else {
+        setExpandedNodes((oldExpanded) =>
+          oldExpanded.filter((item) => item != nodeId),
+        );
+        action(`Closed node ${nodeId}`)();
+      }
+    },
+    [],
+  );
+
+  const [disabledNodes] = React.useState([
+    'Node 4',
+    'Node 3.0',
+    'Node 6',
+    'Node 10',
+  ]);
+
+  const generateItem = React.useCallback(
+    (index: number, parentNode = '', depth = 0): StoryData => {
+      const keyValue = parentNode ? `${parentNode}.${index}` : `${index}`;
+      return {
+        id: `Node ${keyValue}`,
+        label: `Node ${keyValue}`,
+        subLabel: `Sublabel for Node ${keyValue}`,
+        subItems:
+          depth < 10
+            ? Array(Math.round(index % 5))
+                .fill(null)
+                .map((_, index) => generateItem(index, keyValue, depth + 1))
+            : [],
+      };
+    },
+    [],
+  );
+
+  const data = React.useMemo(
+    () =>
+      Array(50)
+        .fill(null)
+        .map((_, index) => generateItem(index)),
+    [generateItem],
+  );
+
+  const getNode = React.useCallback(
+    (node: StoryData): NodeData<StoryData> => {
+      return {
+        subNodes: node.subItems,
+        nodeId: node.id,
+        node: node,
+        isExpanded: expandedNodes.findIndex((id) => id === node.id) != -1,
+        isDisabled: disabledNodes.findIndex((id) => id === node.id) != -1,
+        isSelected: selectedNodes.findIndex((id) => id === node.id) != -1,
+        hasSubNodes: node.subItems.length > 0,
+      };
+    },
+    [disabledNodes, expandedNodes, selectedNodes],
+  );
+
+  return (
+    <Tree<StoryData>
+      data={data}
+      getNode={getNode}
+      nodeRenderer={React.useCallback(
+        ({ node, ...rest }) => (
+          <TreeNode
+            label={node.label}
+            sublabel={node.subLabel}
```

Comment:
so `sublabel` or `subLabel`. I guess first one?

Reply:
Changed to sublabel